dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -182,7 +183,6 @@ def load_user
     # TODO: This should use cancan/authorize.
     if user.student_of?(current_user)
       @user = user
-      @user_level = @user.user_level_for(@script_level)
```

Comment:
It's not clear to me why it's safe to move this declaration into a logically very distinct part of `load_level_source`; can you explain?